build-system/tasks/firebase.js

```diff
@@ -47,12 +47,17 @@ async function copyAndReplaceUrls(src, dest) {
   await Promise.all(promises);
 }
 
-async function modifyThirdPartyUrl() {
-  const filePath = 'firebase/dist/amp.js';
-  const data = await fs.readFile('firebase/dist/amp.js', 'utf8');
+/**
+ * Adds the Firebase host to the prepended AMP_CONFIG in order to make ads
+ * work on deployed demos.
+ * @param {string} filePath
+ */
+async function modifyThirdPartyUrl(filePath) {
+  const data = await fs.readFile(filePath, 'utf8');
   const result = data.replace(
     'self.AMP_CONFIG={',
-    'self.AMP_CONFIG={"thirdPartyUrl":location.origin,'
+    // eslint-disable-next-line prettier/prettier
+    'self.AMP_CONFIG={"thirdPartyUrl":location.origin,"thirdPartyFrameRegex": "^.*\.firebaseapp\.com$",'
```

Comment:
I believe you need a double-backslash here for a single-backslash to appear in the `AMP_CONFIG`.
```
'self.AMP_CONFIG={"thirdPartyUrl":location.origin,"thirdPartyFrameRegex": "^.*\\.firebaseapp\\.com$",'
```
If that works, the `eslint-disable-next-line prettier/prettier` in the line above will no longer be necessary.

Reply:
Based on a new approach, I've actually removed this code altogether. Let me know if this approach works better.